src/main/java/androidx/text/SpannableStringBuilder.kt

```diff
@@ -134,3 +138,39 @@ inline fun SpannableStringBuilder.scale(
     proportion: Float,
     builderAction: SpannableStringBuilder.() -> Unit
 ) = inSpans(RelativeSizeSpan(proportion), builderAction = builderAction)
+
+/**
+ * Wrap appended text in [builderAction] in a [SuperscriptSpan].
+ *
+ * @see SpannableStringBuilder.inSpans
+ */
+inline fun SpannableStringBuilder.up(builderAction: SpannableStringBuilder.() -> Unit) =
+    inSpans(SuperscriptSpan(), builderAction = builderAction)
+
+/**
+ * Wrap appended text in [builderAction] in a [SubscriptSpan].
+ *
+ * @see SpannableStringBuilder.inSpans
+ */
+inline fun SpannableStringBuilder.down(builderAction: SpannableStringBuilder.() -> Unit) =
```

Comment:
I don't think "up" and "down" correctly convey the behavior. "superscript" and "subscript" seem fine.

Also are these common enough that they warrant inclusion? I can't remember using them once.